corehq/apps/registration/views.py

```diff
@@ -476,7 +476,7 @@ def confirm_domain(request, guid=''):
         track_confirmed_account_on_hubspot.delay(requesting_user)
         request.session['CONFIRM'] = True
 
-        if settings.IS_SAAS_ENVIRONMENT:
+        if settings.IS_SAAS_ENVIRONMENT and not settings.ENABLE_NEW_TRIAL_EXPERIENCE:
             # For AppCues v3, land new user in Web Apps
```

Comment:
since users can only sign up for community, new users should always land on the dashboard.

Reply:
Will there be a followup - or a PR into this one - to remove the rest of AppCues v3?

Reply:
I haven't thought about that, but will add it to the list